src/constants/colors.js

```diff
@@ -0,0 +1,12 @@
+const colors = {
+  craftOrange: "#E3976E",
```

Comment:
Glad were switching to storing our color values like this! This will be much better to access and update down the road if we need to.